src/main/java/bio/terra/testrunner/runner/TestRunnerEnvironmentScriptResult.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.testrunner.runner;
+
+public class TestRunnerEnvironmentScriptResult {
+  public String githubRunId;
+  public String githubRepository;
+  public String githubServerUrl;
+
```

Comment:
I'm kind of confused by the class name here. It seems like `TestRunnerEnvironmentScriptResult` would be generic, but this has all the Github specific fields, so regardless of which environment script a user specifies in their server file they're expected to populate exactly these three fields.

Reply:
The intention is for `TestRunnerEnvironmentScriptResult` to capture all runtime context data, not just GitHub runtime context but of course now we only populate github-related context because that's what we focus at the moment.